app/konnect/dedicated-cloud-gateways.md

```diff
@@ -0,0 +1,63 @@
+
+Dedicated Cloud Gateways also have the following benefits:
+* SOC1 compliant out-of-the-box
+* {{site.konnect_short_name}} handles gateway upgrades for you
+* Supported on the following AWS regions: Sydney, Tokyo, Singapore, Frankfurt, Ireland, London, Ohio, Oregon
+* Choose public or private mode to decide who can view your APIs. In public mode, powered by Kong's public Edge DNS for clusters, you can expose your APIs to the internet. 
+
+You can manage your Dedicated Cloud Gateway nodes in [Gateway Manager](https://cloud.konghq.com/gateway-manager/).
+
+![cloud gateway dashboard](/assets/images/products/konnect/gateway-manager/konnect-control-plane-cloud-gateway.png)
+> _**Figure 1:** Example of the dedicated cloud gateway data plane node dashboard in Gateway Manager. The dashboard displays the requests, error rate, and P99 latency. It also displays the cluster configuration, as well as the data plane groups and regions._
+
+Dedicated Cloud Gateways are useful for the following use cases:
+
+| Use case | Cloud gateway solution |
+| ------- | ----------- |
+| Reducing latency is important to your org. You need an infrastructure that is fault tolerant and has low-latency. | Dedicated Cloud Gateways are supported on multiple AWS regions: Sydney, Tokyo, Singapore, Frankfurt, Ireland, London, Ohio, Oregon. This helps you achieve high availability and regional failover so that if one region goes down, you can switch to another. You can use this capability to implement configurations like [cross-region DNS-based load balancing and failover](https://docs.aws.amazon.com/whitepapers/latest/real-time-communication-on-aws/cross-region-dns-based-load-balancing-and-failover.html). |
+| Your organization operates in a regulated industry with strict data protection and privacy requirements. | Dedicated Cloud Gateways help ensure compliance by keeping data traffic within a secure, private network, reducing exposure to external threats. If you select the private gateway option, Kong provisions a private network load balancer and only exposes the IP address in the UI. Only your users can hit the private IPs to access the gateway.|
+| Your organization needs high availability with zero downtime while upgrading data plane nodes. | {{site.konnect_short_name}} handles upgrades for you. There's no downtime when upgrading the infrastructure because when Kong performs the rolling upgrade, we synchronize the data plane node with load balancer registration and deregistration and gracefully terminate the old data plane nodes to reduce the impact on ongoing traffic. Additionally, you can pre-warm your cluster by specifying the number of requests per second so that the first requests don’t have to wait for the infrastructure to scale up. |
```

Comment:
```suggestion
| Your organization needs high availability with zero downtime when upgrading data plane nodes. | There's no downtime when upgrading your data plane nodes. Additionally, you can pre-warm your cluster by specifying the number of requests per second so that the first requests don’t have to wait for the infrastructure to scale up. |
```
Maybe link to the upgrade doc?